.cirrus.yml

```diff
@@ -242,3 +242,21 @@ address_sanitizer_task:
     build_script: make -j3 test/test
     test_script: cd test && ./test --order rand
     before_cache_script: *before_cache_script
+
+depslist_task:
+    name: Dependencies list is up to date
+
+    container:
+      cpu: 1
+      memory: 256MB
+      dockerfile: docker/ubuntu_18.04-build-tools.dockerfile
+      docker_arguments:
+          cxx_package: g++-9
+          cc: gcc-9
+          cxx: g++-9
+
+    script:
+        - make depslist
+          # --exit-code forces git-diff to exit with code 1 if there were
+          # changes; that'll fail the task, which is exactly what we need
+        - git diff --exit-code || echo 'WARNING: the above diff is produced by GCC. Copy it if you only have Clang installed; otherwise just run `make depslist` and commit the result'
```

Comment:
Doesn't this just ignore the error because of the `||`?

Reply:
Oh, [it does](https://cirrus-ci.com/task/6490859793285120?command=main#L148). Thanks for spotting that; I'll submit another PR with a fix.

Reply:
#787 should fix that.